src/app/tree/tree-demo.service.ts

```diff
@@ -1,7 +1,7 @@
+
+import {of as observableOf,  Observable } from 'rxjs';
```

Comment:
ah, I think I found the answer to my question above.

Reply:
Yeah - they removed it - not sure why I like the other way.  I should really fix the others aliases too.